src/pages/verify/e-ids/danish-mitid.mdx

```diff
@@ -107,9 +107,12 @@ Controlled Transfers require that you partner with each service provider that yo
 </Highlight>
 
 To enable Controlled Transfer for a session, you must provide `login_hint=dk:mitid:ct` for the initial authentication.
-This will add an additional claim, `mitid:ct:sid` in the JWT we issue once the user is authenticated.
+This will add two additional claims, `mitid:ct:sid` and `mitid:ct:sid:exp`, in the JWT we issue once the user is authenticated.
 The `mitid:ct:sid` claim contains a session token which allows you to initiate Controlled Transfers.
 You must store this token securely.
+The `mitid:ct:sid:exp` claim contains the time when the session token expires.
+After this time, it is no longer possible to perform a Controlled Transfer for the session.
+We suggest that you instead use the ReAuthentication flow to transfer the user if the session token has expired.
```

Comment:
May benefit from some additional line breaks:

(Perhaps a blank lige before the new section, to add a new paragraph)

![image](https://github.com/user-attachments/assets/d81b0e7b-fd13-41b5-970a-35a0a6eddf57)

Reply:
I added some line breaks in f71ad2f.